distributions/README.md

```diff
@@ -0,0 +1,19 @@
+# Criteria for Supported Distributions
+
+Support for a distribution implies all the duties covered by the Approver and Maintainer role requirements. In addition, support means that the Collector SIG is the owner and maintainer of the binaries/images of the different Collector distributions and is responsible for the pipeline that produces those artifacts.
+
+Distributions supported by the Collector SIG should fulfill the following criteria:
+
+1. Support at least one distribution that is recommended for production which includes support for Prometheus, Jaeger, Zipkin, and OpenCensus.
```

Comment:
This one is a bit unclear. It doesn't seem like a criterion for distributions but rather a requirement for the Collector SIG

Reply:
That's fair, I could move it out of the list and just state it as a thing we must remember to do.

Reply:
Moved